tests/test_scale.py

```diff
@@ -45,7 +45,6 @@ def test_scale_air_pressure_to_height():
             ref_height_m=2,
```

Comment:
can you undo any empty line change made for this test_scale.py? we should have two empty lines between functions